tests/r/description-file/verify

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env Rscript
+library('binderdescription')
```

Comment:
I think the travis failure is because this file isn't marked as executable.

Reply:
doh! Let's see whether this'll fix it!